functions/src/services/database/collections.ts

```diff
@@ -98,4 +98,12 @@ export class CollectionsService {
       .collection('scores')
       .withConverter(new DatabaseConverter(scoreConverter))
   }
+
+  userHealthObservations(userId: string, collectionName: string) {
```

Comment:
Now you have `userHealthObservations` with a regular string and `userObservations` where you have an enum to defined for all the available collections, but ultimately they work the same way - could you please remove one of them?